### PR TITLE
MLPAB-1637 Add the other data to the lpa store submission

### PR DESCRIPTION
**Pull request by hawx**
Also see https://github.com/ministryofjustice/opg-data-lpa-store/pull/72

### CODE REVIEW

File: internal/lpastore/client_test.go [hunk 1]
```diff
@@ -241,9 +279,11 @@ func TestClientServiceContract(t *testing.T) {
 				"X-Jwt-Authorization": dsl.Regex("Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJvcGcucG9hcy5tYWtlcmVnaXN0ZXIiLCJzdWIiOiJ0b2RvIiwiaWF0Ijo5NDY3NzEyMDB9.teh381oIhucqUD3EhBTaaBTLFI1O2FOWGe-44Ftk0LY", "Bearer .+"),
 			},
 			Body: dsl.Like(map[string]any{
```
Comment (acsauk) [inline internal/lpastore/client_test.go:281]: Should this include `CertificateProvider`?

Comment (acsauk) [reply internal/lpastore/client_test.go:281]: And is it worth including `PeopleToNotify` or are we going for minimum required data?

**Review by acsauk — COMMENTED** [review]
From looking at the API spec it looks like we need `certificateProvider` as part of the `initialLpa` unless I'm misreading the validation blocks

File: internal/lpastore/client_test.go (hunk 1 above)
Comment (hawx) [reply internal/lpastore/client_test.go:281]: good points

**Pull request merged**